alibi_detect/saving/schemas.py

```diff
@@ -295,7 +295,7 @@ class PreprocessConfig(CustomBaseModel):
     Optional tokenizer for text drift. Either a string referencing a HuggingFace tokenizer model name, or a
     :class:`~alibi_detect.utils.schemas.TokenizerConfig`.
     """
-    device: Optional[Literal['cpu', 'cuda']] = None
+    device: Optional[Literal['cpu', 'cuda', 'gpu']] = None
```

Comment:
Content of this pending decision on https://github.com/SeldonIO/alibi-detect/pull/829/files#r1262574030